var/spack/repos/builtin/packages/gobject-introspection/package.py

```diff
@@ -23,6 +23,7 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+from spack import spack_root
```

Comment:
If I do not include the second import then this happens:

```
==> Error: NameError: global name 'spack_root' is not defined                                                             /home/hartzelg/tmp/spack-gtkplus/var/spack/repos/builtin/packages/gobject-introspection/package.py:78, in setup_environment:
     77       def setup_environment(self, spack_env, run_env):
  >> 78           spack_env.set('SPACK_SBANG', "%s/bin/sbang" % spack_root )
```

I think that `spack_root` is not listed in `__all__` in `lib/spack/spack/__init__.py` (but my python-fu is not magnificent...).  Would it be better to touch this up there (and if so, should I just add it to the list at line #172 or ...?) or to include the second import here?